server/src/rpc.rs

```diff
@@ -565,7 +569,12 @@ impl ChiselRpc for RpcService {
         &self,
         _request: Request<StatusRequest>,
     ) -> Result<Response<StatusResponse>, Status> {
+        let server_id = {
+            let state = self.state.lock().await;
+            state.id.to_simple().to_string()
```

Comment:
Why not use the default Display implementation?
```suggestion
            state.id.to_string()
```